### PR TITLE
support bloom for cpp

### PR DESCRIPTION
**Pull request by a32543254**
## Type of Change

feature or bug fix or documentation or others
API changed or not

## Description

detail description 
JIRA ticket: xxx

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

**Review by zhentaoyu — COMMENTED** [review]
update name-arch table here: https://github.com/intel/intel-extension-for-transformers/blob/main/intel_extension_for_transformers/llm/runtime/graph/models/model_utils/model_types.h#L336-L338

File: intel_extension_for_transformers/llm/runtime/graph/scripts/convert_bloom.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+                    if ftype == 1 else torch.float32, low_cpu_mem_usage=True, trust_remote_code=True)
+    print("Model loaded: ", dir_model)
+
+    fout = open(fname_out, "wb")
+    fout.write(struct.pack("i", 0x67676d6c)) # magic: ggml in hex
+
+    fout.write(struct.pack("i", hparams["vocab_size"]))
+    fout.write(struct.pack("i", hparams["hidden_size"]))
+    fout.write(struct.pack("i", 1))
+    fout.write(struct.pack("i", hparams["n_head"]))
```
Comment (zhentaoyu) [inline intel_extension_for_transformers/llm/runtime/graph/scripts/convert_bloom.py:82]: add `n_head_kv` in line 83

Comment (a32543254) [reply intel_extension_for_transformers/llm/runtime/graph/scripts/convert_bloom.py:82]: fixed in https://github.com/intel/intel-extension-for-transformers/pull/207/commits/470d1758b49b1b04f81b768f52fdd7e057482aba

**Review by zhenwei-intel — APPROVED** [review]
LGTM

**Review by zhentaoyu — APPROVED** [review]
The .cpp files look good to me.  Do you compare the logits (the first token) between the torch and cpp graph?
